public/js/lib/devtools-sham/shared/DevToolsUtils.js

```diff
@@ -92,74 +92,6 @@ exports.makeInfallible = function makeInfallible(aHandler, aName) {
     }
   }
 }
-/**
```

Comment:
moved these functions to `util/utils` as there wasn't a similar function in `DevToolsUtils` also, they're not devtool specific